lib/data_update_scripts/20210630041322_migrate_data_to_work_field.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class MigrateDataToWorkField
+    QUERY = "data->>'employment_title' <> '' AND data->'work' IS NULL".freeze
```

Comment:
The `data->>'employment_title' <> ''` takes care of both `null` and empty strings.